docs/Using_the_EasyBuild_command_line.rst

```diff
@@ -31,7 +31,7 @@ Whenever EasyBuild searches for easyconfig files, it considers a couple of locat
       (which is included in the default robot search path)
 
 These locations are only considered for easyconfig files that are specified only by filename or using a relative path,
-*not* for easyconfig files that are specified via an absolute path.
+*not* for easyconfig files that are specified via an absolute path. The dependencies are resolved using the robot search path and the path to easyconfig files provided by the EasyBuild installation.
```

Comment:
drop the last part of the sentence (`and the path to`), since that's exactly the default robot search path

maybe include a pointer to the section on robot search path, using:

```
(see :ref:`robot_search_path`)
```